app/Annotations/OpenApi/controllersAnnotations/studentCollaborationAnnotations/AnnotationsStudentCollaboration.php

```diff
@@ -2,50 +2,44 @@
 
 namespace App\Annotations\OpenApi\controllersAnnotations\studentCollaborationAnnotations;
 
-use OpenApi\Annotations as OA;
-
 class AnnotationsStudentCollaboration
 {
-    /**
-     * @OA\Get(
-     *      path="/api/v1/studentCollaborations",
-     *      operationId="getStudentCollaborations",
-     *      tags={"Collaborations"},
-     *      summary="Get collaborations for a student",
-     *      description="Returns a list of collaborations for a specific student. (Note: This endpoint returns hardcoded student collaborations)",
-     *
-     *      @OA\Response(
-     *          response=200,
-     *          description="Successful operation",
-     *          @OA\JsonContent(
-     *              type="array",
-     *              @OA\Items(
-     *                  type="object",
-     *
-     *                  @OA\Property(
-     *                      property="collaborations",
-     *                      type="array",
-     *                      @OA\Items(
-     *                          type="object",
-     *                          @OA\Property(property="id", type="integer", example=1, description="Collaboration ID"),
-     *                          @OA\Property(property="name", type="string", example="ita wiki", description="Collaboration name"),
-     *                          @OA\Property(property="description", type="string", example="Recursos subidos", description="Collaboration description"),
-     *                          @OA\Property(property="quantity", type="integer", example=9, description="Quantity"),
-     *                      ),
-     *                      
-     *                  ),
-     *
-     *              )
-     *          )
-     *      ),
-     *      @OA\Response(
-     *          response=404,
-     *          description="Student not found"
-     *      )
-     * )
-     */
-    public function __invoke()
-    {
-    }
-}
+/**
+ * @OA\Get(
+ *     path="/students/{uuid}/collaborations",
+ *     operationId="getStudentCollaborations",
+ *     summary="Retrieve a list of collaborations",
+ *     tags={"Collaboration"},
+ *
+ *
+ *          @OA\Parameter(
+ *          name="uuid",
+ *          description="Student UUID",
+ *          required=true,
+ *          in="path",
+ *          @OA\Schema(
+ *              type="string",
+ *              format="uuid"
+ *          )
+ *      ),
+ *
+ *     @OA\Response(
+ *         response=200,
+ *         description="Successful. Collaboration list retrieved",
+ *         @OA\JsonContent(
+ *             type="array",
+ *                 @OA\Items(
+ *                     type="object",
+ *                     @OA\Property(property="uuid", type="string", format="uuid"),
+ *                     @OA\Property(property="collaboration_name", type="string"),
+ *                     @OA\Property(property="collaboration_description", type="text"),
```

Comment:
I think the type of this property should be ``string``.

Even though this filld is a text type in the database, it seems that Swagger doesn't recognize the text type, and considering that the endpoint will actually return a string, ``type="string"`` would be more appropriate.